ros/src/waypoint_updater/waypnt_updater.py

```diff
@@ -346,6 +351,9 @@ def traffic_cb(self, traffic_msg):
             rospy.logdebug("same /traffic_waypoint message tl_wp = {} received."
                            .format(traffic_msg.data))
 
+    def dbw_enabled_cb(self, dbw_enabled_message):
+        self.dbw_enabled = dbw_enabled_message.data
+
```

Comment:
I think we should add a logwarn here if the status changes.
```
        if self.dbw_enabled != dbw_enabled_message.data:
            rospy.logwarn("Setting dbw to {} in waypoint_updater".format(dbw_enabled_message.data))
            self.dbw_enabled = dbw_enabled_message.data
```

Reply:
Good call, I'll add this before merging.